### ISSUE TITLE
naming issues with sbref

### ISSUE THREAD

**Issue by oricon** [jmtyszka/bidskit#28]
I have singleband reference images (one per multiband func run) that I'm trying to convert, but I'm running into naming trouble. I've setup Protocol_Translator as follows:
```
"ritl_pa_32ch_mb8_v01_r01_SBRef":[
        "func",
        "task-ritl_acq-SB_run-01_sbref",
        "task-ritl_acq-MB_run-01_bold"
    ],
    "ritl_pa_32ch_mb8_v01_r01":[
        "func",
        "task-ritl_acq-MB_run-01_bold",
        "UNASSIGNED"
    ],
```
But they are being named as if the sbref are duplicates of the bold images:
sub-M80301092_task-ritl_acq-MB_run-01_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-MB_run-02_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-MB_run-03_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-MB_run-04_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-MB_run-05_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-MB_run-06_run-01_bold.nii.gz
sub-M80301092_task-ritl_acq-SB_run-01_run-02_sbref.nii.gz
sub-M80301092_task-ritl_acq-SB_run-02_run-02_sbref.nii.gz
sub-M80301092_task-ritl_acq-SB_run-03_run-02_sbref.nii.gz
sub-M80301092_task-ritl_acq-SB_run-04_run-02_sbref.nii.gz
sub-M80301092_task-ritl_acq-SB_run-05_run-02_sbref.nii.gz
sub-M80301092_task-ritl_acq-SB_run-06_run-02_sbref.nii.gz

I've tried w/o the IntendedFor field and I get the same result. Is this a bug/issue or am I defining them incorrectly?

Thanks,
Joe

**Comment by jmtyszka** [comment]
Hi Joe - this is a bug. I have time this week to work on this and the other active issues, so I'll hopefully have a fix for you to try out by next week. Thanks for raising this.

**Comment by jmtyszka** [comment]
I've just pushed a new version (v1.1.0) which fixes this bug. The duplicate series counting logic has been updated and now correctly recognizes the difference between a BOLD EPI timeseries and its associated SBRef (auto generated by the CMRR MB sequence). If a run-* value is present in the protocol translator BIDS filename template, this is preserved and not overwritten. If a series description occurs only once in the session, no run-* field is added to the BIDS filename suffix.
